src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -1,35 +1,48 @@
 import React, { useCallback, useMemo } from "react";
 import { useHistory } from "react-router-dom";
+import { useCss } from "react-use";
+import { faUserFriends } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import classNames from "classnames";
 
 import { PosterPageVenue } from "types/venues";
 
 import { WithId } from "utils/id";
-import { enterSpace, externalUrlAdditionalProps } from "utils/url";
+import { enterSpace } from "utils/url";
 
 import { useWorldParams } from "hooks/worlds/useWorldParams";
 
 import { PosterCategory } from "components/atoms/PosterCategory";
 
 import "./PosterPreview.scss";
 
+interface PosterAttendanceProps {
+  userCount: number;
+}
+
+const PosterAttendance: React.FC<PosterAttendanceProps> = ({ userCount }) => {
```

Comment:
Looks like this could use a separate file `PosterAttendance.tsx`

Reply:
Resolved in https://github.com/sparkletown/sparkle/pull/2777/commits/9065fb9c7cb99b420e55cdda05bdc1a2c9844c70